src/ensembl/src/shared/components/help-popup/HelpPopupBody.scss

```diff
@@ -60,3 +53,13 @@ $heading-outdent: 16px;
     fill: $grey;
   }
 }
+
+.aside {
+  grid-column: aside;
+
+  h2 {
+    font-size: 13px;
+    font-weight: $light;
+    margin: 1.5rem 0;
+  }
+}
```

Comment:
Huh? These styles are reused in the Help app, which is why they were in HelpArticle.scss. Moving them out breaks styles in Help app.